src/TipBox.vue

```diff
@@ -185,11 +209,11 @@
     .icon-wrapper {
         display: flex;
         margin-right: .5em;
-        width: 22px;
-        height: 22px;
     }
 
     .contents {
+        display: flex;
+        align-items: center;
```

Comment:
Why are these changes necessary?

Reply:
This was added to vertically center-align the content of the box.
Here's what the content will look like without the vertical alignment -  
<img width="828" alt="Screenshot 2020-01-23 at 9 37 59 PM" src="https://user-images.githubusercontent.com/21968718/72989148-a79e3a00-3e28-11ea-8857-6ca20f28a83e.png">

Vertically aligned content -
<img width="811" alt="Screenshot 2020-01-23 at 9 41 06 PM" src="https://user-images.githubusercontent.com/21968718/72989374-167b9300-3e29-11ea-8e95-e6fc6881de62.png">

Reply:
Did you try with long paragraphs? I don't think these changes are appropriate.

Reply:
Test for different length content to make sure the solution work for all situations (in general, it's safer to use realistic data for examples/tests). Use fontawsome icons when giving screenshots.

Reply:
> Did you try with long paragraphs? I don't think these changes are appropriate.

Yup, you're right. This was causing problems for paragraphs. I've modified the css class for proper vertical center aligment now and tested it out with large paragraphs and single-line content.

Tests with different icon sizes -
1. small icon; single-line 
<img width="809" alt="Screenshot 2020-01-23 at 11 40 01 PM" src="https://user-images.githubusercontent.com/21968718/72999018-b3dec300-3e39-11ea-903d-ea9810cd992a.png">

* medium icon; single-line
<img width="809" alt="Screenshot 2020-01-23 at 11 44 08 PM" src="https://user-images.githubusercontent.com/21968718/72999384-45e6cb80-3e3a-11ea-88ac-8bddcb6ead74.png">

* large icon; single-line
<img width="810" alt="Screenshot 2020-01-23 at 11 44 45 PM" src="https://user-images.githubusercontent.com/21968718/72999452-5c8d2280-3e3a-11ea-818e-25e0ae6e7124.png">

* extra-large icon; single-line
<img width="806" alt="Screenshot 2020-01-23 at 11 45 17 PM" src="https://user-images.githubusercontent.com/21968718/72999520-6f9ff280-3e3a-11ea-815f-bed2684aba31.png">

* medium icon; multi-line
<img width="805" alt="Screenshot 2020-01-24 at 12 04 26 AM" src="https://user-images.githubusercontent.com/21968718/73001240-1c7b6f00-3e3d-11ea-9276-5b6f1c4b41a3.png">

* large icon; multi-line
<img width="804" alt="Screenshot 2020-01-24 at 12 05 11 AM" src="https://user-images.githubusercontent.com/21968718/73001312-374de380-3e3d-11ea-88d1-beb24ead546a.png">

* extra-large icon; multi-line
<img width="805" alt="Screenshot 2020-01-24 at 12 05 29 AM" src="https://user-images.githubusercontent.com/21968718/73001339-42a10f00-3e3d-11ea-9413-9e9ae49bdc7c.png">

Reply:
Looks OK to me.